src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -459,6 +460,20 @@ class CCC_SND_Restart : public IConsole_Command
 };
 
 //-----------------------------------------------------------------------
+float ps_sun_direction = 0.0f;
+class CCC_SunDirection : public CCC_Float
+{
+public:
+    CCC_SunDirection(LPCSTR N, float* V) : CCC_Float(N, V, 0.0f, 360.0f) {}
+    virtual void Execute(LPCSTR args)
+    {
+        CCC_Float::Execute(args);
+        if (g_pGamePersistent)
+            g_pGamePersistent->Environment().m_sun_azimuth = ps_sun_direction * (PI / 180.0f);
```

Comment:
Can you test what happens when `r__sun_direction 180` is in the user.ltx file? Will the sun direction still be corrected?